protos/manual_control/manual_control.proto

```diff
@@ -0,0 +1,85 @@
+syntax = "proto3";
+
+package mavsdk.rpc.manual_control;
+
+option java_package = "io.mavsdk.manual_control";
+option java_outer_classname = "ManualControlProto";
+
+service ManualControlService
+{
+
+    
+    /*
+     * Send command to enable Joystick mode 
+     *
+     * Enabling Joystic mode causes to change into Position Control mode
+     * drone still needs to be armed
+     * Needs to check if the drone is not currently flying or in mission if so drone needs to be paused first 
+     */
+    
+    rpc StartPositionControl (PositionControlRequest)  returns(PositionControlResponse) {}
```

Comment:
@julianoes: Didn't we say that setting the flight mode should be an action?

Reply:
Either that or this at least for now.

I'm reluctant to add a flight mode setter before we have proper MAVLink flight modes.

Reply:
should I keep it or delete it ?

Reply:
Let's keep it for now